jenkins/Jenkinsfile

```diff
@@ -109,12 +109,13 @@ pipeline {
                       pip install cloudify==5.1.0
                       cfy profile use ${env.OS_IP} -u ${env.OS_USERNAME} -p ${env.OS_PASSWORD} -t ${env.OS_TENANT}
                       pushd 'bp'
-                        cfy install -b ec2-py27-blueprint ec2-py27-blueprint.yaml
+                        cfy install -b ec2-py27-blueprint-${env.BUILD_NUMBER} ec2-py27-blueprint.yaml
```

Comment:
why we using a dynamic name for blueprint? it may cause a situation when you overload cloudify manager on RS with multiple blueprints and not deleting them when you finished

Reply:
the question if want this?

Reply:
It was done to prevent a clash if more than 1 build is running from the multibranch, causing the build to fail since the blueprint is already installed.
The deletion of will be done even if something fails in the stage or other stages.
That's the best way that I found, do you have any other suggestions to it?

Reply:
got you, parallel builds problem